controller.py

```diff
@@ -74,6 +95,31 @@ def handle_add_me_to_queue_callback(
                 callback_query["id"]
             )
 
+    def handle_cross_out_next_callback(
+            self,
+            callback_query,
+            callback_query_data
+    ):
+        try:
+            queue_id = callback_query_data["queue_id"]
+            error = self.repository.cross_out_next(queue_id)
```

Comment:
I think it would be better to split the `cross_out_next` function into two, one for getting the next queue member and another one for crossing it out. That way we can make the logic, and specifically error handling, a bit easier to understand